assets/css/general.css

```diff
@@ -38,44 +48,101 @@ h2 {
     margin-top: 10px;
 }
 
-/* Utilities */
+html[dir='ltr'] .d-flex .flex-wrap > *:not(*:last-child) { /* sass-lint:disable-line class-name-format */
+    font-size: smaller;
+    margin-right: 3px;
+}
 
-/* sass-lint:disable class-name-format */
-.position-relative {
-    position: relative;
+html[dir='rtl'] .d-flex .flex-wrap > *:not(*:first-child) { /* sass-lint:disable-line class-name-format */
+    font-size: smaller;
+    margin-right: 3px;
 }
 
-.d-none {
-    display: none;
+html[dir='ltr'] .otherTranslationOptions > *:not(*:last-child) { /* sass-lint:disable-line class-name-format */
+    margin-right: 5px;
 }
 
-.d-inline-block {
-    display: inline-block;
+html[dir='rtl'] .otherTranslationOptions > *:not(*:first-child) { /* sass-lint:disable-line class-name-format */
```

Comment:
Why does this need `class-name-format` disabled? Same concern re. general selectors here.

Reply:
Oops, My fault, Added for all. Gotta remove.

Reply:
Done. :)